### PR TITLE
Establish_websocket_backend

### PR DESCRIPTION
**Pull request by Peders123**

### CODE REVIEW

**Review by Peders123 — COMMENTED** [review]
Established the initial technology required for backend websocket communication.

Please approve this request if you think it's okay, but don't merge it. I would like to review it myself and create follow-up tickets.

**Comment by Peders123** [comment]
[Establish websocket backend](https://trello.com/c/zUgGquXI/8-establish-websocket-backend)

**Review by revilokid — APPROVED** [review]
looks good

**Pull request closed without merging**
